CONTRIBUTING.md

```diff
@@ -317,6 +323,19 @@ Note that for VSCode, to enable "live" linting of TypeScript (and other) file ty
     ]
 ```
 
+To make `prettier` automatically format your code on save, open add those lines below:
+
+```json
```

Comment:
I think we'll want to remove this and go eslint first.  eslint can fix, and includes prettier + more rules